NetKAN/ProceduralFairings.netkan

```diff
@@ -1,16 +1,6 @@
 {
-    "spec_version": 1,
+    "spec_version": "v1.4",
     "identifier": "ProceduralFairings",
-    "$kref": "#/ckan/github/rsparkyc/ProceduralFairings",
-    "$vref": "#/ckan/ksp-avc",
-    "x_netkan_epoch": 1,
```

Comment:
It's not safe to drop `x_netkan_epoch` properties. Newer releases will be treated as older.

Reply:
Are you sure you want the epoch in the NetKAN repo instead of in your metanetkan? Either will work, but putting it in your metanetkan would give more control.